documentation/reference/document-json-format.html

```diff
@@ -905,11 +915,40 @@ <h3 id="add-weighted-set">Add weighted set entries</h3>
 </pre>
 </p>
 
+<h3 id="tensor-add">Add tensor cells</h3>
+<p>
+Add cells to the mapped dimension of tensors. Invalid for tensors with only indexed
+dimensions. Adding a cell that already exists will overwrite the cell value with the new value.
+
+<pre>
+field tensorfield type tensor(x{},y[3]) {
```

Comment:
I think we should change the type to tensor(x{},y[2]) such that the example specifies the entire indexed sub-space.

Reply:
Actually, the idea here was to show default values in add, but that point was missing. Fixed now.